tuna/miopen/benchmark.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+###############################################################################
+#
+# MIT License
+#
+# Copyright (c) 2022 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+###############################################################################
+""" Module for defining benchmark and model enums """
+
+import enum
+from sqlalchemy import Column, Integer, String, UniqueConstraint, ForeignKey, DateTime
+from sqlalchemy import Enum
+from sqlalchemy import Float
+from tuna.dbBase.base_class import BASE
+
+
+class FrameworkEnum(enum.Enum):
+  """Represents framework enums"""
+  PYTORCH = 'Pytorch'
+  DEEPBENCH = 'Deepbench'
+
+
+class Framework(BASE):
+  """Represents framework table"""
+  __tablename__ = "framework"
+  __table_args__ = (UniqueConstraint("framework", name="uq_idx"),)
+  framework = Column(Enum(FrameworkEnum), nullable=False)
+
+
+class ModelEnum(enum.Enum):
+  """Represents model enums"""
+  RESNET50 = 'Resnet50'
```

Comment:
@Dmantri98 can you help me add more frameworks and models here that MIOpen looks at?

Reply:
@JehandadKhan  im adding: mobilenet, ssd, lstm? Anything else?

Reply:
Lets start with these and add more later